scss/inc/_functions.scss

```diff
@@ -48,7 +48,13 @@ Usage:
     $spanPercent: widthPerc($spanPx, $gridPx);
     $marginPercent: widthPerc($gutter, $gridPx);
     margin-left: $marginPercent;
-    width: $spanPercent;
+    inline-size: $spanPercent;
+}
+@mixin grid-unit-writing-mode-margin($span, $numCols: 12, $gutter: 0) {
+    $gridPx: 840;
+    $marginPercent: widthPerc($gutter, $gridPx);
+    margin-inline-start: $marginPercent;
+    margin-left: 0;
```

Comment:
Is it ok that this (and the `margin-left` on line 50) are not converted to `inline/block` variants?